modules/ROOT/pages/backup-and-disaster-recovery.adoc

```diff
@@ -95,6 +97,58 @@ Outputs logs to a file.
 
 . Manually restore the NFS files if you are not reusing the original NFS.
 
+== Anypoint Monitoring and Visualizer
```

Comment:
Please add a note that this only applies when `Anypoint Monitoring and Visualizer` is enabled.

Reply:
Added.